### ISSUE TITLE
Remove "snap-both" class from sheet

### ISSUE THREAD

**Issue by ttowntom** [NoroffFEU/yeetsheet#145]
The class creates a bug in Firefox

**Comment by sulenchy** [comment]
Can you add more info about the bug on Firefox? You can add some screenshots and procedures on how to recreate the bug. @ttowntom

**Comment by ttowntom** [comment]
> Can you add more info about the bug on Firefox? You can add some screenshots and procedures on how to recreate the bug. @ttowntom

Certainly!

**Recreate the bug:**

1. Keep the snap-both class applied
2. Open the app i Firefox
3. Scroll down the sheet, then up again

Screenshot of bug:
![image](https://github.com/user-attachments/assets/7a4a7d27-1c00-41fb-849a-31030ea55364)

@Bakek-lang worked on this bug previously and is the best person to ask for further details.

**Comment by Barvand** [comment]
@whamre Hi Werner, 
Do you have an update on this?

**Comment by Barvand** [comment]
This ticket has been reassigned to @Glenn-2k as the ticket has to be completed before Friday.

**Comment by Barvand** [comment]
Has been resolved in another ticket by Team Elg.
